src/store/wizardSlice.ts

```diff
@@ -438,6 +438,11 @@ export const wizardSlice = createSlice({
         1
       );
     },
+    clearOscapPackages: (state) => {
+      state.packages = state.packages.filter(
+        (pkg) => pkg.repository !== 'openscap'
```

Comment:
I’m a little wary of using the `repository` field to keep track of whether a package is included due to OpenSCAP or not… it just feels a confusing to me because it isn’t _technically_ true – the actual repository the package would get installed from would still be the `distro` repo, right?.

What if we add a field to `IBPackagewithRepositoryInfo` that is specifically related to whether the package is being added due to an OpenSCAP profile? Something like `isRequiredByOpenScapProfile`? And I suppose it would be a boolean. 

That's just one idea, there might be a better way to do it. But my main point is that I think we should keep track of this in a new field in the object.

Also a bit of refactoring I’d like to request that's not really related to this PR but would be nice – right now `repository` is a string. Can we create a new type for this? Just something like `type PackageRepository = 'distro' | ...`.

Reply:
Good point, I've decoupled the oscap info from the repository. The added field it is!

The `PackageRepository` type is also added with the possible values being 'distro', 'custom' and an empty uninitialized string.

Reply:
Awesome, thank you!